tests/conftest.py

```
@@ -0,0 +1,129 @@
+    Create example CAN Addressing Information.
+
+    :param addressing_format: CAN Addressing Format to use.
+
+    :return: Example CAN Addressing Information object using provided CAN Addressing Format.
+    """
+    if addressing_format == CanAddressingFormat.NORMAL_ADDRESSING:
+        return NormalCanAddressingInformation(rx_physical_params={"can_id": 0x720},
+                                              tx_physical_params={"can_id": 0x748},
+                                              rx_functional_params={"can_id": 0x7DF},
+                                              tx_functional_params={"can_id": 0x748})
+    elif addressing_format == CanAddressingFormat.NORMAL_FIXED_ADDRESSING:
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)